app/views/site/how_it_works_newbie.erb

```diff
@@ -106,18 +106,11 @@
   <div class="row">
     <div class="col-md-8">
       <p>
-      This might be the trickiest part of Exercism.
-      </p>
-
-      <p>
       How you install the Exercism CLI depends on what kind of computer you have.
-      We've explained all of this in detail on the <a href="/cli">CLI page</a>, but
-      it's pretty confusing.
+      We've explained all of this in detail on the installation page (link, below).
```

Comment:
I paired with @traceydolby04 on this (last night's Fullstack LA).  As she walked through the onboarding process I silently observed her scanning for what to do next.  She confirmed it was unclear.

I've been wondering what that was.  I think there are at least 2 phenomena at play:

1. people don't read the web (yours truly included).
2. a link _within_ a body of text tends to convey it is a source of clarification (e.g. definition of a term or process) whereas a link _standing alone_ tends to convey a call to action.

This may seem like a nitpicky stance, but I observed how it injected additional cognitive load in a process that's already (admittedly!) sometimes bumpy for folks.

By the way, the CLI install went really smoothly for Tracey.  She was already familiar with `brew` and instantly recognized what was going on, once she got to the install instructions.